include/tiramisu/core.h

```diff
@@ -4021,6 +4024,50 @@ class computation
     virtual void skew(int i, int j, int k, int l, int factor);
 
 
+    /*
+      this perform a general polyhedral transformation i' = a*i +b*j under det(A)=1 constraint for j' : b!=0 & a >=0
+  
+     */
+
+    /**
+      * 
+      * 
+      * Apply a general polyhedral transformation on the loop levels \p i and \p j . 
+      *  and replaced with new var j' i' such as :
+      * i' = a*i +b*j 
+      * guarentees that the automatically computed j' creates a new iteration domain where loop increment is always =1
+      * additional constraints for inputs are :  b!=0 & a >=0
+      *           a & b must be prime between themselfs
+      * This command transforms the loop (i, j) into the loop (a*i+b*j,j').
+      * For example if you have the following loop
+      *
+      * \code
+      for (int i = 1; i < N; i++)
+        for (int j = 1; j < M; j++)
+          a[i][j] = a[i - 1][j] + a[i][j - 1];
+       \endcode
+
+      * and apply
+
+      \code
+        a.skew(i, j, 1, 1, ni, nj);
+      \endcode
+
+      * you would get
+      *
+      \code
+      for (int i = 1; i < 2N; i++)
+        for (int j = max(i,N-i); j < Min(i,N) ; j++)
+          a[i-j][j] = a[i - j - 1][j] + a[i-j][j - 1];
+      \endcode
+
+      */
+
+    virtual void skewing(var i, var j, int f_i , int f_j, var ni, var nj);
+
+    virtual void skewing(int i , int j , int f_i , int f_j ); 
```

Comment:
There a syntax of 

@{
...
...
@}

that we usually use for functions that overload each other (like in this example). Just check for the syntax in functions that have the same name. This syntax instructs doxygen to use the same documentation for both functions. One of the functions will not have a very accurate documentation but that is Ok because people will just look at the other overloaded function.